src/SDKs/CognitiveServices/dataPlane/Language/LUIS/Runtime/Microsoft.Azure.CognitiveServices.LUIS.Runtime.csproj

```diff
@@ -6,15 +6,16 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.CognitiveServices.Language.LUIS.Runtime</PackageId>
     <Description>Provides API functions for consuming the Microsoft Azure Cognitive Services LUIS Runtime API.</Description>
-    <Version>1.0.0</Version>
+    <Version>2.0.0</Version>
     <AssemblyName>Microsoft.Azure.CognitiveServices.Language.LUIS</AssemblyName>
     <PackageTags>Microsoft Cognitive Services;Cognitive Services;Cognitive Services SDK;LUIS;Language Understanding Intelligent Service;Language Understanding;REST HTTP client</PackageTags>
     <PackageReleaseNotes>
     <![CDATA[
     This is a public release of the Cognitive Services LUIS Runtime SDK.
 
     Changes in this release:
-    1. Support customizing service endpoints. 
+    1. Support customizing service endpoints. The endpoint string can be found on Azure Portal.
```

Comment:
Suggest to use below description: (The reason is we may not change Azure Portal quickly, so better in here (as well as documents) we emphasize that the Endpoint should only contains protocol and hostname)

1. Supported customizing service endpoints by assigning the endpoint string to LUISRuntimeClient.Endpoint. The endpoint string can be found on Azure Portal, it should contain only protocol and hostname, for example: https://westus.api.cognitive.microsoft.com.
2....